lib/ansible/modules/known_hosts.py

```diff
@@ -40,6 +40,7 @@
   path:
     description:
       - The known_hosts file to edit.
+      - The path to the known_hosts file must exist prior to running the module.
```

Comment:
we might want to rephrase, as we will create the file

The file will be created if needed but the rest of the path must exist prior to running the module.